### PR TITLE
Front End Pull Request

### PR DESCRIPTION
**Pull request by JayAlvinCVallescas**
Add new feature in front end

### CODE REVIEW

File: package-lock.json
Comment (H-isaac23) [inline package-lock.json:1]: delete to hahaha

**Review by H-isaac23 — CHANGES_REQUESTED** [review]
Delete lang package-lock.json

File: package-lock.json
Comment (H-isaac23) [inline package-lock.json:1]: hoiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii andito pa rin hahaha

**Review by H-isaac23 — CHANGES_REQUESTED** [review]
delete package-lock.json aaaaaaaaaaaaaaaaaaaaaaaaaaa

**Review by H-isaac23 — APPROVED** [review]
lgtm

**Comment by JayAlvinCVallescas** [comment]
Okkiii

**Pull request merged**